tf2/src/cache.cpp

```diff
@@ -60,6 +60,8 @@ TimeCache::TimeCache(ros::Duration max_storage_time)
 : max_storage_time_(max_storage_time)
 {}
 
```

Comment:
I closed the namespace immediately below the declarations so the using tf2 is used through the rest of the file. Do you think it's cleaner to just put the tf2 namespace at the top instead of "using".